core/src/test/scala/org/apache/spark/sql/delta/ConvertToDeltaSQLSuite.scala

```diff
@@ -16,20 +16,47 @@
 
 package org.apache.spark.sql.delta
 
+import org.apache.spark.sql.delta.sources.DeltaSQLConf
 import org.apache.spark.sql.delta.test.DeltaSQLCommandTest
+import org.apache.spark.sql.functions.{col, from_json}
 
 trait ConvertToDeltaSQLSuiteBase extends ConvertToDeltaSuiteBaseCommons
   with DeltaSQLCommandTest {
   override protected def convertToDelta(
       identifier: String,
-      partitionSchema: Option[String] = None): Unit = {
+      partitionSchema: Option[String] = None, collectStats: Boolean = true): Unit = {
```

Comment:
are all the existing tests now run with `collectStats=true`?

Reply:
Yes, except from one under SQLSuite that checks false